modules/packages/Python.chpl

```diff
@@ -405,6 +410,8 @@ module Python {
 
       Note: this only works with lambdas that accept a tuple of arguments,
       like 'x,' or '*args'
+
+      Creates a new reference
```

Comment:
Not as far as I can tell it doesn't, there's no incref in the body.

Reply:
`PyRun_String` returns a new reference and is not decrefed in this function

Reply:
github hid a big chunk of the diff from me :)